### DISCUSSION TITLE
Landmarks as additional input channels for guidance possible?

### DISCUSSION THREAD

**Discussion by che85** [MIC-DKFZ/nnUNet#1934] [General]
Hi, 

you have done an amazing job here! Thank you for that!

I was curious if there is a way to use landmarks (e.g. distance maps) or landmarks that get transformed into distance maps and use those as additional input channels?

I noticed that if I use labels as additional input channels, the preprocessing step calculates mean values of these labels, which doesn't make sense.

Thank you in advance.

Christian

**Comment by che85** [comment]
@FabianIsensee Any thoughts on this topic?

**Comment by FabianIsensee** [comment]
Hi,
I am not keen on writing additional code to include that as this will add additional complexity for a quite specific use case. What I can envisage doing in the future is allow adding optional information about the input channel so that they don't get augmented with intensity augmentations.
Right now, you should already be able to do what you want to do. Here is how:
- Landmarks as segmentation maps
  - append them as additional modality to your input
  - use one-hot encoding
  - declade the modality in dataset.json as 'noNorm' (will disable normalization)
- Landmarks as distance transform
  - add as additional modality
  - declare modality as CT to avoid problems with initialization and/or normalization

Best,
Fabian

**Reply by che85** [reply]
@FabianIsensee That's already very helpful and I will give it a try. Thank you for your response.
